tgui/packages/tgui/components/Dropdown.tsx

```diff
@@ -143,7 +143,9 @@ export function Dropdown(props: Props) {
       scrollTo(selectedIndex);
     }
 
-    innerRef.current?.focus();
+    innerRef.current?.addEventListener('load', () => {
+      innerRef.current?.focus();
+    });
```

Comment:
I'm not seeing any benefit from this change. Probably not necessary unless you saw an improvement. I should probably just explore what TG has done to Popper and Dropdown since the parity port. E.g. https://github.com/tgstation/tgstation/pull/80556